packages/netlify-cms-core/src/actions/mediaLibrary.js

```diff
@@ -253,6 +253,7 @@ export function loadMediaDisplayURL(file) {
     if (typeof url === 'string' || typeof displayURL === 'string') {
       dispatch(mediaDisplayURLRequest(id));
       dispatch(mediaDisplayURLSuccess(id, displayURL));
+      return;
```

Comment:
BUG: Looks like a bug introduced in https://github.com/netlify/netlify-cms/pull/2430

This `return` was removed as a part of the pr.

Without it you can see the following errors in the `console` when trying to load the media libary:

`Error: getMediaDisplayURL is not implemented by the current backend, but the backend returned a displayURL which was not a string!`

As a result no images are displayed.

Reply:
Fixed in #2523: https://github.com/netlify/netlify-cms/pull/2523/commits/d42d27480d4ba2db923a4101a4f0a86eb9bb2f49